### PR TITLE
Create a blog to keep track of upcoming kubeCon EU activities related to Istio

### PR DESCRIPTION
**Pull request by linsun**
@craigbox in the project video, you can also build a QR for this page as well to show all Istio related activities in kubecon EU.

Will update the blog as the schedule is updated.

### CODE REVIEW

File: content/en/blog/2024/kubecon-eu/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+There will be lots of Istio-related activity at KubeCon + CloudNativeCon Europe in Paris! We'll keep this page updated with more details as they are published.
+
+{{< image width="75%"
+    link="./KubeConParis.png"
+    alt="KubeCon + CloudNativeCon Europe, March 19-22, 2024, Paris, France. #kubecon"
+    >}}
+
+- Come to the [IstioDay](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/istio-day/) co-located event
```
Comment (dhawton) [inline content/en/blog/2024/kubecon-eu/index.md:16]:
```suggestion
- Come to the [Istio Day](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/istio-day/) co-located event
```

File: content/en/blog/2024/kubecon-eu/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- Attend the co-located ObservabilityDay session related to Istio: [A Practical Guide on How to Monitor and Compare Service Mesh Infrastructure Costs - Lin Sun, Solo.io](https://colocatedeventseu2024.sched.com/event/1YFf2/cl-lightning-talk-a-practical-guide-on-how-to-monitor-and-compare-service-mesh-infrastructure-costs-lin-sun-soloio?iframe=no)
```
Comment (dhawton) [inline content/en/blog/2024/kubecon-eu/index.md:18]:
```suggestion
- Attend the co-located Observability Day session related to Istio: [A Practical Guide on How to Monitor and Compare Service Mesh Infrastructure Costs - Lin Sun, Solo.io](https://colocatedeventseu2024.sched.com/event/1YFf2/cl-lightning-talk-a-practical-guide-on-how-to-monitor-and-compare-service-mesh-infrastructure-costs-lin-sun-soloio?iframe=no)
```

**Review by dhawton — COMMENTED** [review]
Quick pass

**Comment by linsun** [comment]
> Quick pass. Some of the names will need to get added to the .spelling file.

Thanks Daniel! I'll watch out for make gen output.

**Review by dhawton — APPROVED** [review]
LGTM

**Pull request merged**